linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -86,6 +86,14 @@ public static BinaryExpression add(Expression left, Expression right,
     throw Extensions.todo();
   }
 
+  /**
+   * Creates a BinaryExpression that represents an arithmetic
+   * mod operation that does not have overflow checking.
```

Comment:
I was curious on how `mod` can cause overflow, so searched a bit and found: https://stackoverflow.com/questions/19285163/does-modulus-overflow

Just FYI, if there is a need to check overflow, the link proposes a way

Reply:
revise doc.